### PR TITLE
PLT-3050 Fixed /join error

### PR DESCRIPTION
**Pull request by DavidLu1997**
Fixed error upon /join, now users can join open channels without errors!

### CODE REVIEW

File: api/command_join.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ func (me *JoinProvider) DoCommand(c *Context, channelId string, message string)
 				if v.Type != model.CHANNEL_OPEN {
 					return &model.CommandResponse{Text: c.T("api.command_join.fail.app_error"), ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL}
 				}
-				return &model.CommandResponse{GotoLocation: c.GetTeamURL() + "/channels/" + v.Name, Text: c.T("api.command_join.success"), ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL}
+				return &model.CommandResponse{GotoLocation: v.Name + "/channels/" + v.Id, Text: c.T("api.command_join.success"), ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL}
```
Comment (jwilander) [inline api/command_join.go]: This looks incorrect, why is the channel ID added to the URL?

Comment (jwilander) [inline api/command_join.go]: Also it's supposed to be `<teamname>/channels/<channelname>` when you have `<channelname>/channels/<channelid>`

Comment (DavidLu1997) [inline api/command_join.go]: That is intentional, otherwise client side handling becomes much more difficult (asynchronous calls to `getMoreChannels()`)

Comment (jwilander) [inline api/command_join.go]: This is a big hack and not the way to fix this issue. I also can't reproduce the bug on `master`

**Comment by DavidLu1997** [comment]
Closing because fixed somewhere else

**Pull request closed without merging**